setup.cfg

```diff
@@ -17,6 +17,7 @@ include_package_data = True
 install_requires = flake8
                    junit_xml
                    mock
+		   defusedxml
```

Comment:
Am I seeing this indent right? Is there a tab in there?

Reply:
/me checks vimrc copied over from the old laptop and fixes the issue both there and in the commit